### ISSUE TITLE
Fix Accessibility of Code Block

### ISSUE THREAD

**Issue by jcfilben** [grommet/grommet-site#361]
If a user is navigating by keyboard and tabbing through elements on a page. If the page has an editable code block a keyboard user will get 'trapped' within the code block. Once they tab into the code block they are unable to leave by pressing 'shift-tab' or 'tab'. We need to evaluate a better way to handle editable code blocks for users that navigate with a keyboard.
